ygot/struct_validation_map_test.go

```diff
@@ -52,6 +55,142 @@ func errToString(err error) string {
 	return err.Error()
 }
 
+func TestStructTagToLibPaths(t *testing.T) {
```

Comment:
Suggest adding tests specifically for gnmiPath type. This splits up the logic specific to that type vs. tag to lib paths code. Several of the tests below could be moved into that test function.

Reply:
Done.